logstash-core/lib/logstash/instrument/collector.rb

```diff
@@ -19,7 +19,7 @@ class Collector
     include Observable
     include Singleton
 
-    SNAPSHOT_ROTATION_TIME_SECS = 10 # seconds
+    SNAPSHOT_ROTATION_TIME_SECS = 1 # seconds
```

Comment:
I know, this is still work in progress, but why this change again? I mean we changed from 1 to 10 and not back to 1, what is your thinking here?